templates/layouts/custom-gift-amount.liquid

```diff
@@ -131,14 +117,28 @@
       _.pick(champaign.personalization.location, 'currency')
     );
 
+    var supportedCountryLanguageList = ['en-US', 'en-GB', 'en-CA', 'en-NZ', 'en-AU', 'en-IE', 'de-DE', 'de-AT', 'de-CH', 'fr-FR', 'fr-CA', 'es-ES', 'es-AR', 'es-MX'];
+    var countryLanguage = `${champaign.page.language_code}-${champaign.personalization.location.country}`;
+    supportedCountryLanguageList.includes(countryLanguage)
+      ? null
+      : countryLanguage = `${champaign.page.language_code}`
```

Comment:
Oh one last thing: since this code is in the template and does not get transpiled, we can't use template strings as they're not supported in IE11 (which unfortunately we support!).